tools/Makefile.sub

```diff
@@ -8,10 +8,15 @@ jlc_SOURCES = \
 jlc_LIBS += \
 	libtooling \
 	libllvm \
+	libmlir \
 	librvsdg \
 	libutil \
 
-jlc_EXTRA_LDFLAGS += $(shell $(LLVMCONFIG) --libs core irReader) $(shell $(LLVMCONFIG) --ldflags) $(shell $(LLVMCONFIG) --system-libs)
+jlc_EXTRA_LDFLAGS += \
+	       $(shell $(LLVMCONFIG) --libs core irReader) \
+	       $(shell $(LLVMCONFIG) --ldflags) \
+	       $(shell $(LLVMCONFIG) --system-libs) \
```

Comment:
Could we shrink this into one command similarly to how it is in jlm/tooling/Makefile.sub

Reply:
All flags given to a single LLVMCONFIG